### PR TITLE
Carps, frogs and young lobstrosities now fear the fishing hat!

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Carps, frogs and young lobstrosities now fear legendary anglers wearing the legendary fishing hat and will flee. The item is skill-locked, so only those that have maxxed out the skill can wear it.

Differently, adult lobstrosities and megacarps (and suicide frogs, which are used nowhere) do not flee but will still prioritize them over the rest.

## Why It's Good For The Game
I'm putting some emphasis on the "fish fear me" _(lobstrosities and frogs aren't exactly fish but it'd been quite boring to only include carps)_ and also making the hat a bit more than just some cosmetic novelty.

## Changelog

:cl:
add: Carps, frogs and young lobstrosities now fear people wearing fishing hats! Adults and megacarp favour the 'fight' part of the fear reflex however.
fix: The hat stabilizer module now inherits the clothing traits of the attached hat.
/:cl:

### CODE REVIEW

**Comment by Jacquerel** [comment]
Maybe the CL should say "adults and megacarp favour the 'fight' part of the fear reflex".

**Comment by Metekillot** [comment]
What effect will this have on women?

**Comment by Dawnseer** [comment]
Negative moodlet, they also fear the legendary angler

**Comment by Loafin34** [comment]
megacarp should flee, that's too funny

**Comment by Ghommie** [comment]
No, this only affect the ai controllers of these mobs. If you can come up with an idea for sentient mobs, you're welcome.

File: code/datums/ai/basic_mobs/basic_subtrees/flee_target.dm [hunk 1]
```diff
@@ -12,15 +12,30 @@
 	var/atom/flee_from = controller.blackboard[target_key]
 	if (controller.blackboard[BB_BASIC_MOB_STOP_FLEEING] || QDELETED(flee_from))
```
Comment (Ghommie) [inline code/datums/ai/basic_mobs/basic_subtrees/flee_target.dm:13]: fuck, I forgot to remove this line!

**Pull request merged**